src/tye/InitHost.cs

```diff
@@ -58,10 +55,7 @@ public static (string, string) CreateTyeFileContent(FileInfo? path, bool force)
             if (path is FileInfo && path.Exists)
             {
                 var application = ConfigFactory.FromFile(path);
-                var serializer = new SerializerBuilder()
-                    .WithNamingConvention(CamelCaseNamingConvention.Instance)
-                    .ConfigureDefaultValuesHandling(DefaultValuesHandling.OmitDefaults)
-                    .Build();
+                var serializer = YamlSerializer.CreateSerializer();
```

Comment:
Hm, seems like the tye init tests are failing due to a new line difference (Expected \r\n, got \n). Not sure why that would change. 

I added a work item to do tests with yaml comparison rather than file comparison as new line differences like these aren't important to test.

Reply:
I recently fill a issue about this https://github.com/dotnet/tye/issues/235

Reply:
May as well do it in this change since it's failing tests

Reply:
ok

Reply:
done!!